go.mod

```diff
@@ -31,10 +31,16 @@ require (
 	github.com/uber/jaeger-client-go v2.30.0+incompatible
 	github.com/uber/jaeger-lib v2.4.1+incompatible
 	golang.org/x/net v0.0.0-20221019024206-cb67ada4b0ad
-	google.golang.org/grpc v1.50.1
+	google.golang.org/grpc v1.51.0
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+replace (
```

Comment:
Create a branch on `gopowerstore` and `gobrick` repositories and point the changes to that commit instead of having it as a local reference here.

Reply:
Done